tools/Repo-Tasks.psm1

```diff
@@ -7,6 +7,8 @@ $userPsFileDir = [string]::Empty
 Function Init()
 {
     #Initialize Code
+	#Import-Module $PSScriptRoot\Modules\TestFx-Tasks.psm1
```

Comment:
I don't think we need to do this, as the modules are already listed as depednecies in the module manifest (.psd1)